configure.ac

```diff
@@ -51,21 +51,38 @@ AM_MISSING_PROG([AUTOM4TE], [autom4te])
 # Needed by tests/atlocal.in.
 AC_SUBST([O0CFLAGS], [`echo $CFLAGS | sed 's/-O[[0-9]] *//'`])
 
-AM_PATH_PYTHON
-AM_CONDITIONAL([HAVE_PYTHON], test -n "$PYTHON")
-[if test -z "$PYTHON"; then]
-    [echo "The python interpreter was not found in the search path. The Python language bindings"]
-    [echo "will not be built.  If you want to build the Python language bindings, please ensure"]
-    [echo "that python is installed and its directory is included in the search path."]
-    [echo "Then run configure again before attempting to build Satyr."]
-[fi]
+AC_ARG_WITH([python2],
+            [AS_HELP_STRING([--without-python2],
+                            [Build without python2 support.])],
+            [with_python2=$withval],
+            [with_python2=yes])
+AM_CONDITIONAL(WITH_PYTHON2, [test $with_python2 = yes])
+[if test "$with_python2" = "yes"]
+[then]
+    AC_PATH_PROG([PYTHON], [python], [no])
+    [if test "$PYTHON" = "no"]
+    [then]
+        [echo "The python program was not found in the search path. Please ensure"]
+        [echo "that it is installed and its directory is included in the search path or"]
+        [echo "pass --without-python2 to ./configure."]
```

Comment:
```$ ./configure --without-python2``` ends with error for me 
```
configure: error: in `/home/mhabrnal/satyr':
configure: error: The pkg-config script could not be found or is too old.  Make sure it
is in your PATH or set the PKG_CONFIG environment variable to the full
path to pkg-config.

Alternatively, you may set the environment variables RPM_CFLAGS
and RPM_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```